docs/standard/serialization/serialization-samples-binary.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
when I run SEO tests on CATS, it complains when we have duplicate words on the path like serialization here shows up twice.
I'm not going after fixing the migrated samples, but I try to follow that rule when creating new articles.

Reply:
These articles were created by adding -xml and -binary to the existing file name.  I can rename them in a separate PR before master goes to live.